vector-app/src/gplay/java/im/vector/app/push/fcm/GoogleFcmHelper.kt

```diff
@@ -56,10 +56,9 @@ class GoogleFcmHelper @Inject constructor(
         }
     }
 
-    override fun ensureFcmTokenIsRetrieved(activity: Activity, pushersManager: PushersManager, registerPusher: Boolean) {
-        //        if (TextUtils.isEmpty(getFcmToken(activity))) {
+    override fun ensureFcmTokenIsRetrieved(pushersManager: PushersManager, registerPusher: Boolean) {
         // 'app should always check the device for a compatible Google Play services APK before accessing Google Play services features'
-        if (checkPlayServices(activity)) {
+        if (checkPlayServices(context)) {
```

Comment:
From what I understood, we only use the checking method [GoogleApiAvailability.isGooglePlayServicesAvailable(context)](https://developers.google.com/android/reference/com/google/android/gms/common/GoogleApiAvailability#public-int-isgoogleplayservicesavailable-context-context). This method only requires a `Context`. In case of error, we display a toast which only requires a `Context` as well.